src/trex/cli/run10x.py

```diff
@@ -96,6 +99,8 @@ def main(args):
             amplicon_inputs=amplicon_inputs,
             sample_names=sample_names,
             prefix=args.prefix,
+            min_bases_detected=args.min_bases_detected,
+            per_cell_correction=args.per_cell,
```

Comment:
It’s more maintainable in the long run to use the exact same names for the `run_trex` parameter and the `args` attribute. So perhaps
```
correct_per_cell=args.correct_per_cell
```
The flag can still be named `--per-cell`, but you would need to add a `dest="correct_per_cell"` to the `add_argument()` call.